src/GoogleFit/FitOnFhir.GoogleFit/Client/Handlers/GoogleFitAuthorizationHandler.cs

```diff
@@ -148,5 +158,13 @@ private async Task<IActionResult> Revoke(RoutingRequest request, AuthState state
                 return new ObjectResult("An unexpected error occurred while attempting to revoke data access.") { StatusCode = StatusCodes.Status500InternalServerError };
```

Comment:
Now that we have a "Global" try/catch in the RoutingService, we should remove all of the try/catch blocks that return a 500. This will clean up the code quite a bit.